documentation/docs/20-core-concepts/30-form-actions.md

```diff
@@ -329,6 +329,8 @@ The easiest way to progressively enhance a form is to add the `use:enhance` acti
 +<form method="POST" use:enhance>
 ```
 
+> `use:enhance` can only be used with forms that have `method="POST"`. It will not work with `method="GET"`, which includes forms without a specified method. Attempting to use `use:enhance` on forms without `method="POST"` will result in an error.
```

Comment:
```suggestion
> `use:enhance` can only be used with forms that have `method="POST"`. It will not work with `method="GET"`, which is the default for forms without a specified method. Attempting to use `use:enhance` on forms without `method="POST"` will result in an error.
```